extensions/notebook/src/book/remoteBook.ts

```diff
@@ -8,7 +8,7 @@ import * as utils from '../common/utils';
 import { IAsset } from './remoteBookController';
 
 export abstract class RemoteBook {
-	protected _localPath: URL;
+	protected _localPath: vscode.Uri;
 
 	constructor(public remotePath: URL, public outputChannel: vscode.OutputChannel, protected _asset?: IAsset) {
```

Comment:
You're still using URL here - shouldn't this be updated to Uri as well?

Reply:
The issue only happens with local paths, but I think you are right. Maybe we should replace completely the other module if we are using vscode uri, instead